utils/docker/examples/README.md

```diff
@@ -0,0 +1,441 @@
+- `DAOS_DOCKER_IMAGE_TAG`: Tag identifier of the DAOS client docker image (default "rocky8.6")
+- `DAOS_VERSION`: Version of DAOS to use (default "2.4.0-2.el8")
+
+When the environment file has been properly filled, the docker image could be created thanks to the
+following command:
+```bash
+docker compose --file utils/docker/examples/docker-compose.client_sa.yml build daos_agent
+```
+
+### Running DAOS Docker Images
+
+This section presents how to run some relevant use cases with a docker image build according to the
+previous section.  In a first time the following environment variables of the docker environment
+file `utils/docker/examples/.env` must be defined:
+- `DAOS_CLIENT_UID`: User id of the client (e.g.  "0")
+- `DAOS_CLIENT_GID`: Group id of the client (e.g. "0")
+
+In a second time, a tarball (i.e. `tar` archive compressed with `xz`) of the DAOS certificate files
+should be created when the DAOS authentication is enabled.  For using Docker Compose the tarball of
+the certificates file path should be readable by all users and its file path defined in the
+following variable of the docker environment file `utils/docker/examples/.env`:
+- `DAOS_AGENT_CERTS_TXZ`: tarball containing the DAOS certificated needed by the DAOS agent
+  (e.g. "secrets/daos\_agent-certs.txz")
+
+!!! note
+    As [Docker Secret](https://docs.docker.com/engine/swarm/secrets/) is a Docker Swarm service, it
+    could not be used properly with Docker Compose.  With Docker Compose, secret are managed as
+    standard Docker Volume mounted in `/run/secrets` directory.  More details could be found at:
+    https://github.com/docker/compose/issues/9139#issuecomment-1098137264
+
+!!! note
+    For properly managing secret, Docker Stack should be used instead of Docker Compose.  Sadly, the
+    `pid=host` option is not yet supported in swarm mode, and this last one is mandatory to allow
+    the DAOS Agent to monitor its associated clients. More details could be found at:
+    https://github.com/docker/docs/issues/5624 and https://github.com/moby/swarmkit/issues/1605
+
+When the environment file has been properly filled, then an application such as `daos pool autotest`
+could be run in the following way:
+```bash
+docker compose --file utils/docker/examples/docker-compose.client_sa.yml up --detach daos_agent
+docker compose --file utils/docker/examples/docker-compose.client_sa.yml run --rm daos_client
+$ daos pool autotest <POOL NAME>
+```
+
+With the same prerequites, the [fio](https://fio.readthedocs.io/) file system benchmark tool could
+be run thanks to the following commands:
+```bash
+docker compose --file utils/docker/examples/docker-compose.client_sa.yml up --detach daos_agent
+docker compose --file utils/docker/examples/docker-compose.client_sa.yml run --rm daos_client
+$ mkdir -p "/home/<DAOS_CLIENT_UNAME>/mnt"
+$ daos container create --type=posix tank posix-fs
+$ dfuse --mountpoint="/home/<DAOS_CLIENT_UNAME>/mnt" --pool=tank --container=posix-fs
```

Comment:
This will work but I think we prefer positional args in documentation?  @daltonbohning could confirm.

Reply:
- [ ] Use positional arguments